docs/reference/function-configuration/function-configuration-reference.md

```diff
@@ -56,35 +60,39 @@ The `spec` section contains the requirements and attributes and has the followin
 | replicas | int | The number of desired instances; 0 for auto-scaling. |
 | minReplicas | int | The minimum number of replicas |
 | maxReplicas | int | The maximum number of replicas |
 | targetCPU | int | Target CPU when auto scaling, as a percentage (default: 75%) |
 | dataBindings | See reference | A map of data sources used by the function ("data bindings") |
 | triggers.(name).maxWorkers | int | The max number of concurrent requests this trigger can process |
-| triggers.(name).kind | string | The kind of trigger. One of `http`, `kafka`, `kinesis`, `eventhub`, `cron`, `nats`, `rabbitmq` |
+| triggers.(name).kind | string | The trigger type (kind) - `cron` \| `eventhub` \| `http` \| `kafka` \| `kinesis` \| `nats` \| `rabbitmq` |
 | triggers.(name).url | string | The trigger specific URL (not used by all triggers) |
 | triggers.(name).annotations | list of strings | Annotations to be assigned to the trigger, if applicable |
 | triggers.(name).workerAvailabilityTimeoutMilliseconds | int | The number of milliseconds to wait for a worker if one is not available. 0 = never wait |
 | triggers.(name).attributes | See [reference](/docs/reference/triggers) | The per-trigger attributes |
-| build.path | string | A local directory or URL to a file/archive containing source and configuration |
-| build.functionSourceCode | string | The source code of the function, encoded in Base64. Mutually exclusive with build.path |
+| <a id="spec.build.path"></a>build.path | string | Function-code GitHub directory or archive-file path for the `github` or `archive` [code-entry type](#spec.build.codeEntryType); see [Code-Entry Types](/docs/reference/function-configuration/code-entry-types.md#external-source-code-entry-types) |
+| <a id="spec.build.functionSourceCode"></a>build.functionSourceCode | string | Base-64 encoded function source code for the `sourceCode` [code-entry type](#spec.build.codeEntryType); see [Code-Entry Types](/docs/reference/function-configuration/code-entry-types.md#code-entry-type-sourcecode) |
 | build.registry | string | The container image repository to which the built image will be pushed |
 | build.noBaseImagePull | string | Do not pull any base images when building, use local images only |
 | build.noCache | string | Do not use any caching when building container images |
-| build.baseImage | string | The base image from which the processor image will be built |
+| build.baseImage | string | The URL of the base image from which the function processor image will be built |
 | build.Commands | list of string | Commands run opaquely as part of container image build |
-| build.onbuildImage | string | The "onbuild" image from which the processor image will be built; can use {{ .Label }} and {{ .Arch }} for formatting |
+| build.onbuildImage | string | The URL of the "onbuild" image from which the function processor image will be built; can include `{{ .Label }}` and `{{ .Arch }}` for formatting |
```

Comment:
same as before, no URL,  just `"onbuild" image` is good

Reply:
I changed the description of `spec.build.onbuildImage` to 'The name of an "onbuild" container image from which to build the function's processor image; the name can include `{{ .Label }}` and `{{ .Arch }}` for formatting'. See commit 837c1bc.